ca_gazebo/src/create_virtual_wall_plugin.cpp

```diff
@@ -119,7 +119,7 @@ void VirtualWallSensorPlugin::OnUpdate()
     const ignition::math::Pose3d & robot_pose = model->WorldPose();
     sensor_offset = robot_pose.Rot().RotateVector(sensor_offset);
     ignition::math::Pose3d sensor_pose = robot_pose + sensor_pose;
-    ROS_INFO_NAMED("virtual_wall_plugin", "sensor_pose x=%f y=%f z=%f",
+    ROS_DEBUG_NAMED("virtual_wall_plugin", "sensor_pose x=%f y=%f z=%f",
```

Comment:
Was this change on purpose?

Reply:
Yes, that is the information of the virtual walls that by default were printing in the console every update, so you can't really see the gazebo startup information because they are quickly displaced by a barrage of messages showing the values of every virtual wall.